packages/itwin/breakdown-trees/src/Views/NodeRenderers/FunctionalTreeNodeRenderer.tsx

```diff
@@ -68,9 +38,16 @@ export const createFunctionalTreeNodeRenderer = (
       className={
         visibilityHandler === undefined ? undefined : "with-checkbox"
       }
-      visibilityHandler={visibilityHandler}
+      visibilityHandler={enableVisibility ? visibilityHandler : undefined}
```

Comment:
did this line need to change? a little redundant

Reply:
Yes, visibilityHandler can have value but we won't pass it here if enableVisibility is false.